bindings/rust/s2n-tls-tokio/tests/send_and_recv.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use s2n_tls_tokio::{TlsAcceptor, TlsConnector};
+use tokio::io::{AsyncReadExt, AsyncWriteExt};
+
+mod common;
+
+const TEST_DATA: &[u8] = "hello world".as_bytes();
+
+#[tokio::test]
+async fn send_and_recv_basic() -> Result<(), Box<dyn std::error::Error>> {
+    let (server_stream, client_stream) = common::get_streams().await?;
+
+    let connector = TlsConnector::new(common::client_config()?.build()?);
+    let acceptor = TlsAcceptor::new(common::server_config()?.build()?);
+
+    let (mut client, mut server) =
+        common::run_negotiate(connector, client_stream, acceptor, server_stream).await?;
+
+    assert_eq!(client.write(TEST_DATA).await?, TEST_DATA.len());
```

Comment:
Might want to use `write_exact` and `read_exact` in case the payloads get fragmented differently by the socket.